contactaccessrights/CRM/Contactaccessrights/BAO/Rights.php

```diff
@@ -1,12 +1,10 @@
 <?php
 
+use CRM_Contactaccessrights_Utils_RightType_RightTypeInterface as RightTypeInterface;
+use CRM_Contactaccessrights_Utils_RightType_Region as RegionRightType;
+use CRM_Contactaccessrights_Utils_RightType_Location as LocationRightType;
+
 class CRM_Contactaccessrights_BAO_Rights extends CRM_Contactaccessrights_DAO_Rights {
-  /**
-   * Rights grouped by type.
-   *
-   * @var array
-   */
-  private $rights = [];
```

Comment:
Why was this removed?

Reply:
It was no longer necessary, The `getRightsByType` method used it to store values in an array initially before returning the result, I rewrote that part not to need this variable anymore.
See: https://github.com/civicrm/civihr/blob/c92225222b9f4fb0461191568883e9bee6df4d24/contactaccessrights/CRM/Contactaccessrights/BAO/Rights.php#L84

Reply:
But why did you rewrite it? Was it causing any problem? Besides, after this refactoring, I don't think the array needs to be indexed by `$entityType` anymore